.php-cs-fixer.dist.php

```diff
@@ -43,7 +43,12 @@
         __DIR__ . '/spark',
     ]);
 
-$overrides = [];
+$overrides = [
+    'php_unit_data_provider_name' => [
+        'prefix' => '',
+        'suffix' => 'Provider',
```

Comment:
I like `prefix (string): prefix that replaces "test"; defaults to 'provide'`.

If so, all test method names begin with `test`, and all data provider names begin with `provide`.

Reply:
So, we'll have `['prefix' => 'provide', 'suffix' => '']`?

Reply:
Yes.